tests/by-util/test_users.rs

```diff
@@ -31,3 +31,12 @@ fn test_users_check_name() {
 
     new_ucmd!().succeeds().stdout_is(&expected);
 }
+
+#[test]
+#[cfg(target_os = "openbsd")]
+fn test_users_check_name_openbsd() {
+    new_ucmd!()
+        .args(&["basic_openbsd_utmp"])
+        .run()
+        .stdout_contains("jadi");
```

Comment:
what means jadi ? :)

Reply:
its my user name. no specific meaning. If not appropriate to use it, I can add a new user to my OpenBSD (say test_user), login and use that utmp file instead.

Reply:
yes, please :)

Reply:
done